tests/gnrc_ipv6_ext_frag/tests-as-root/01-run.py

```diff
@@ -173,10 +173,38 @@ def test_ipv6_ext_frag_shell_test_1(child, s, iface, ll_dst):
     pktbuf_empty(child)
 
 
+def _check_iface(child):
+    # get TAP MAC address
+    child.sendline("ifconfig")
+    ethos_id = None
+    mock_id = None
+    hwaddr = None
+    for _ in range(2):
+        child.expect(r"Iface\s+(\d+)\s+.*")
+        match = re.search(r"HWaddr:\s+([0-9A-F:]{17})\s+",
+                          child.match.group(0))
+        if match:
+            # interface has a hardware address
+            ethos_id = int(child.match.group(1))
+            hwaddr = match.group(1)
+            # consume MTU for later calls of `ifconfig {mock_id}`
+            child.expect(r"MTU:(\d+)")
+        else:
+            mock_id = int(child.match.group(1))
+            # consume MTU for later calls of `ifconfig {mock_id}`
+            child.expect(r"MTU:(\d+)")
```

Comment:
small improvement proposal:

```suggestion
        if match is not None:
            # interface has a hardware address
            ethos_id = int(child.match.group(1))
            hwaddr = match.group(1)
        else:
            mock_id = int(child.match.group(1))
        # consume MTU for later calls of `ifconfig {mock_id}`
        child.expect(r"MTU:(\d+)")
```

Reply:
Taken :-)